numpy/core/tests/test_dtype.py

```diff
@@ -856,8 +856,13 @@ class PackedStructure(ctypes.Structure):
             itemsize=18))
         self.check(PackedStructure, expected)
 
-    @pytest.mark.xfail(sys.byteorder != 'little',
-        reason="non-native endianness does not work - see gh-10533")
+    def test_complex_big_endian_structure_packed(self):
```

Comment:
I'd drop the word "complex" - this is no more complex than the test below it.

Reply:
Done.